src/colmap/sensor/models.h

```diff
@@ -1091,14 +1097,19 @@ void OpenCVFisheyeCameraModel::ImgFromCam(
   ImgFromFisheye(params, uu + duu, vv + dvv, x, y);
 }
 
-void OpenCVFisheyeCameraModel::CamFromImg(
+bool OpenCVFisheyeCameraModel::CamFromImg(
     const double* params, double x, double y, double* u, double* v, double* w) {
-  // Lift points to normalized plane
   double uu, vv;
   FisheyeFromImg(params, x, y, &uu, &vv);
```

Comment:
We could now also handle invalid pixels in the fisheye conversion, but then we should do the equivalent for ImgFromCam... At this point, the induced callsite changes become quite large. Not sure if worth it for a few exotic cameras.